service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -208,6 +208,7 @@
     private final JudgeAndLegalAdvisor judgeAndLegalAdvisor;
     private final C2DocumentBundle temporaryC2Document;
     private final List<Element<C2DocumentBundle>> c2DocumentBundle;
+    private final List<Element<C2DocumentBundle>> furtherEvidenceDocumentsTEMP;
```

Comment:
I feel like you probably don't need this field with the current implementation in CaseData. You only seem to be using it for tests atm.